packages/react/src/components/searchInput/SearchInput.tsx

```diff
@@ -148,24 +154,30 @@ export const SearchInput = <SuggestionItem,>({
           className={classNames(styles.input)}
         />
         <div className={styles.buttons}>
-          {inputValue.length > 0 && (
-            <button
-              type="button"
-              aria-label={clearButtonAriaLabel}
-              className={classNames(styles.button)}
-              onClick={clear}
-            >
-              <IconCrossCircle className={styles.searchIcon} aria-hidden />
-            </button>
-          )}
-          <button
-            type="button"
-            aria-label={searchButtonAriaLabel}
-            className={classNames(styles.button)}
-            onClick={submit}
-          >
-            <IconSearch className={styles.searchIcon} aria-hidden />
-          </button>
+          <>
```

Comment:
This Fragment is probably not needed?

Reply:
- [x] Remove unnecessary React fragments